test/unittests/commands/xmlupload/test_transform_resource_and_values_for_upload.py

```diff
@@ -0,0 +1,425 @@
+        assert prop_name == KNORA_API.hasSegmentBounds
+        val_bn = next(result.objects(res_bn, prop_name))
+        rdf_type = next(result.objects(val_bn, RDF.type))
+        assert rdf_type == KNORA_API.IntervalValue
+        start = next(result.objects(val_bn, KNORA_API.intervalValueHasStart))
+        assert start == Literal("0.1", datatype=XSD.decimal)
+        end = next(result.objects(val_bn, KNORA_API.intervalValueHasEnd))
+        assert end == Literal("0.234", datatype=XSD.decimal)
+
+    def test_segment_of_video(self, lookups: Lookups, res_info: tuple[BNode, str]) -> None:
+        res_bn, _ = res_info
+        res_type = "knora-api:VideoSegment"
+        xml_prop = etree.fromstring("""
+        <isSegmentOf >res_one</isSegmentOf>
+        """)
+        prop = XMLProperty.from_node(xml_prop, "resptr", "onto")
+        result, prop_name = _make_one_prop_graph(prop, res_type, res_bn, lookups)
+        assert len(result) == 3
+        assert prop_name == KNORA_API.isVideoSegmentOfValue
+        val_bn = next(result.objects(res_bn, prop_name))
+        rdf_type = next(result.objects(val_bn, RDF.type))
+        assert rdf_type == KNORA_API.LinkValue
+        value = next(result.objects(val_bn, KNORA_API.linkValueHasTargetIri))
+        assert value == RES_ONE_URI
+
+    def test_segment_of_audio(self, lookups: Lookups, res_info: tuple[BNode, str]) -> None:
+        res_bn, _ = res_info
+        res_type = "knora-api:AudioSegment"
+        xml_prop = etree.fromstring("""
+        <isSegmentOf >res_one</isSegmentOf>
+        """)
+        prop = XMLProperty.from_node(xml_prop, "resptr", "onto")
+        result, prop_name = _make_one_prop_graph(prop, res_type, res_bn, lookups)
+        assert len(result) == 3
+        assert prop_name == KNORA_API.isAudioSegmentOfValue
+        val_bn = next(result.objects(res_bn, prop_name))
+        rdf_type = next(result.objects(val_bn, RDF.type))
+        assert rdf_type == KNORA_API.LinkValue
+        value = next(result.objects(val_bn, KNORA_API.linkValueHasTargetIri))
+        assert value == RES_ONE_URI
+
+
+class TestMakeOnePropGraphRaises:
+    def test_permissions(self, lookups: Lookups, res_info: tuple[BNode, str]) -> None:
+        res_bn, res_type = res_info
+        xml_prop = etree.fromstring("""
+        <integer-prop name=":hasInteger">
+            <integer permissions="nonExistent">4711</integer>
+        </integer-prop>
+        """)
+        prop = XMLProperty.from_node(xml_prop, "integer", "onto")
+        err_str = regex.escape("Could not find permissions for value: nonExistent")
+        with pytest.raises(PermissionNotExistsError, match=err_str):
+            _make_one_prop_graph(prop, res_type, res_bn, lookups)
+
+    def test_unknown_type(self, lookups: Lookups, res_info: tuple[BNode, str]) -> None:
+        res_bn, res_type = res_info
+        xml_prop = etree.fromstring("""
+        <other-prop name=":hasInteger">
+            <other permissions="nonExistent">4711</other>
```

Comment:
There shouldn't be multiple wrong things in 1 test.

```suggestion
            <other>4711</other>
```